client/my-sites/backup/backup-contents-page/file-browser/file-info-card.tsx

```diff
@@ -232,9 +232,11 @@ function FileInfoCard( {
 	// Dispatch an error notice if the file info could not be retrieved
 	useEffect( () => {
 		if ( isError ) {
-			dispatch( onRetrievingFileInfoError() );
+			notices.showError(
+				__( 'There was an error retrieving your file information. Please, try again.' )
```

Comment:
Similar to the comment above:

```suggestion
				__( 'There was an error retrieving your file information. Please try again.' )
```